recipe/meta.yaml

```
@@ -49,14 +49,12 @@ requirements:
     # type checking
     - typeguard >=3,<5
     - imagehash 4.3.1
-    - wordcloud >=1.9.1
+    - wordcloud >=1.9.3
     - dacite >=1.8
-    - numba >=0.56.0,<1
+    - numba >=0.56.0,<=0.61
   run_constrained:
-    - jupyter-client >=5.3.4
-    - jupyter-core >=4.6.3
+    - jupyter >=1.0.0
     - ipywidgets >=7.5.1
```

```suggestion
    - ipywidgets >=7.5.1
    - tangled-up-in-unicode 0.2.0
```
This one is still true.
https://github.com/ydataai/ydata-profiling/blob/v4.16.1/pyproject.toml#L130

We could add the ones of spark. But I would leave it as is for now. As it sets stronger restrictions on visions.